docs/sources/datasources/prometheus/configure-prometheus-data-source.md

```diff
@@ -121,12 +123,14 @@ Following are additional configuration options.
 
 ### Performance
 
-- **Prometheus type** - The type of your Prometheus server. There are four options: `Prometheus`, `Cortex`, `Thanos`, `Mimir`.
+- **Prometheus type** - The type of your Prometheus server. There are four options: `Prometheus`, `Cortex`, `Mimir`, `Thanos`.
 
-- **Version** Select the version you are using. Once the Prometheus type has been selected, a list of versions auto-populates using the Prometheus [buildinfo](https://semver.org/) API. The `Cortex` Prometheus type does not support this API so you will need to manually add the version.
+- **Cache level** - The browser caching level for editor queries. There are four options: `Low`, `Medium`, `High`, `None`.
```

Comment:
I think these should be in bold if they are a UI element perhaps without any formatting: https://grafana.com/docs/writers-toolkit/write/style-guide/style-conventions/#bold
```suggestion
- **Cache level** - The browser caching level for editor queries. There are four options: **Low**, **Medium**, **High**, or **None**.
```

Reply:
Same comment as https://github.com/grafana/grafana/pull/91852#discussion_r1721506356

Reply:
```suggestion
- **Cache level** - The browser caching level for editor queries. There are four options: `Low`, `Medium`, `High`, or `None`.
```